content/docs/data_pipeline/run_through/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-weight: 2
+weight: 3
```

Comment:
Note that you need to change the weightings of all pages on this level, the only reason they still work is because they're sorted by weighting then alphabetical order (presumably).